Make width of "Check updates" button closer to that of the surrounding buttons

gBasil opened the pull request:
Before:
<img width="871" alt="image" src="https://github.com/user-attachments/assets/adf77aba-f0f3-4ec2-910e-35948779c7f8" />

After:
<img width="872" alt="image" src="https://github.com/user-attachments/assets/6736aae1-764b-4a34-b817-0aec91798b24" />

krypciak commented:
this results in a slight overlap between `modOptionsButton` and `checkUpdatesButton` 
![image](https://github.com/user-attachments/assets/2b71aca9-0092-4337-a47f-c8820a9e3264)

krypciak commented:
Wouldn't it be better if the mod settings button stretched to the list end as before?

gBasil commented:
<img width="1132" height="90" alt="image" src="https://github.com/user-attachments/assets/cf73cdb1-1a71-43d0-930b-3410139a4863" />
I've reverted my change and fixed the width (the spacing between the mod options button and the button to the right of it is 10px, instead of the 8px between the other buttons, but I'm not familiar enough with CrossCode's UI system to know why it' doing that).

krypciak commented:
So can this be merged?

gBasil commented:
If you're happy with it, go ahead.
